jdaviz/configs/cubeviz/cubeviz.ipynb

```diff
@@ -10,14 +10,16 @@
     "from jdaviz import Cubeviz\n",
     "\n",
     "cubeviz = Cubeviz(verbosity='JDAVIZ_VERBOSITY', history_verbosity='JDAVIZ_HISTORY_VERBOSITY')\n",
-    "cubeviz.load_data('DATA_FILENAME')\n",
+    "data_path = 'DATA_FILENAME'\n",
+    "if data_path:\n",
+    "    cubeviz.load_data('DATA_FILENAME')\n",
     "cubeviz.app"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.10.7 ('envmain': venv)",
```

Comment:
not sure if this will cause any problems, but might want to clean all the meta-data changes in the notebooks out of the diff?

Reply:
Grr... I wish I could just turn that metadata off my IDE... Thanks for the catch, I'll clean that up